### PR TITLE
001-index

### PR DESCRIPTION
**Pull request by Borys-Menko**

### CODE REVIEW

**Review by 51stRoR — APPROVED** [review]
approved

**Review by stadnytskyivp — APPROVED** [review]
takoe

File: index.html [hunk 1]
```diff
@@ -1,161 +1,2 @@
...
-
-    <script async src="js/alert-window.js"></script>
-    <script defer src="js/products-service.js"></script>
-    <script defer src="js/cart.js"></script>
-    <script defer src="js/product-list.js"></script>    
-    <script defer src="js/index.js"></script>
-</body>
-
-</html>
+Welcom
```
Comment (PavloTytarchuk) [inline index.html]: goose

File: index.html [hunk 2]
```diff
@@ -1,161 +1,2 @@
...
+Home
```
Comment (AndrewRorat) [inline index.html]: I dont like it

**Review by AndrewRorat — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (stadnytskyivp) [inline index.html]: make changes

**Review by PavloTytarchuk — CHANGES_REQUESTED** [review]
look in comments

File: index.html (hunk 1 above)
Comment (stanislavvdoviuk) [inline index.html]: Bad

**Review by stanislavvdoviuk — CHANGES_REQUESTED** [review]

**Review by BorisTitarenko — CHANGES_REQUESTED** [review]
kkk

**Review by 51stRoR — APPROVED** [review]
+

**Review by PavloTytarchuk — APPROVED** [review]

**Review by AndrewRorat — APPROVED** [review]

File: index.html [hunk 3]
```diff
@@ -1,2 +1,2 @@
-Welcom
-Home
\ No newline at end of file
+Welcom Home
```
Comment (stanislavvdoviuk) [inline index.html]: Normalno

**Review by stanislavvdoviuk — APPROVED** [review]

**Review by BorisTitarenko — APPROVED** [review]
nice

**Review by stadnytskyivp — APPROVED** [review]
looks not bad

**Review by LyubomyrKovch — APPROVED** [review]

**Pull request closed without merging**